android/app/src/main/java/org/getlantern/mobilesdk/util/DnsDetector.java

```diff
@@ -61,14 +61,12 @@ public DnsDetector(Context context, String fakeDnsIP) {
                     public void onAvailable(@NonNull Network network) {
                         Logger.debug(TAG, "Adding available network");
                         allNetworks.put(network, "");
-                        EventBus.getDefault().postSticky(Event.NetworkAvailable);
```

Comment:
Not necessary on this PR, but we should just remove the EventBus stuff entirely

Reply:
Yep, eventually we will migrate to use gomodel so most of this kotlin code will be removed.